### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by t-young31**
1. Fixes a bug where 

```
for _ in range(10):
    config = system.configuration()
    config.add_perturbation(0.02, max_length=0.1)
    configs += config.copy()
```

gives the all the same configuration due to the atoms not being copied for each invocation of `system.configuration()`

***

2. Tweaks the default force sigma and bumps the version appropriately
3. Allows for a set of initial fixed configurations to be used

### CODE REVIEW

**Review by tristan-j-wood — COMMENTED** [review]
Looks good

File: gaptrain/active.py [hunk 1]
```diff
@@ -329,7 +330,18 @@ def get_active_configs(config, gap, ref_method_name, method='diff',
     with Pool(processes=int(gt.GTConfig.n_cores)) as pool:
 
         for _ in range(n_configs):
-            result = pool.apply_async(func=function, args=args, kwds=kwargs)
+
+            # Prepend the arguments with the initial configuration
+            if isinstance(config, gt.ConfigurationSet):
+                # Select a configuration at random from the
```
Comment (tristan-j-wood) [inline gaptrain/active.py:336]: from the what?

File: gaptrain/calculators.py [hunk 2]
```diff
@@ -103,8 +103,12 @@ def run_autode(configuration, max_force=None, method=None, n_cores=1, kwds=None)
     except CouldNotGetProperty:
         logger.error('Failed to set forces')
 
-    configuration.energy = ha_to_ev * calc.get_energy()
+    energy = calc.get_energy()
```
Comment (tristan-j-wood) [inline gaptrain/calculators.py:106]: Assume you don't need to do ha_to_ev here anymore?

File: gaptrain/loss.py [hunk 3]
```diff
@@ -103,24 +103,27 @@ def _calculate_single(self, init_config, gap, method_name):
...
-
             logger.info('      ___ |E_true - E_GAP|/eV ___')
             logger.info(f' t/fs      err      cumul(err)')
 
             for j in range(len(traj)):
-                e_error = np.abs(traj[j].energy - pred[j].energy)
+
+                if true[j].energy is None:
+                    logger.warning(f'Frame {j} had no energy')
+                    e_error = 99999999
```
Comment (tristan-j-wood) [inline gaptrain/loss.py:124]: Isn't it better to set this to inf or None?

Comment (t-young31) [reply gaptrain/loss.py:124]: I always forget about `np.inf`. Yes please

**Pull request merged**